stable/sonatype-nexus/Chart.yaml

```diff
@@ -14,3 +14,5 @@ sources:
 maintainers:
   - name: rjkernick
     email: rjkernick@gmail.com
+  - name: vogelinho10
```

Comment:
Please clarify why you added yourself to the maintainers. Are you willing to continue contributing to the chart and reviewing future PRs?

Reply:
Which would be great, of course.